site.js

```diff
@@ -135,6 +135,11 @@ $(document).ready(function (){
         {content: '3.1', start: '2019-01-01', end: '2019-07-01'},
         {content: '3.2', start: '2019-07-01', end: '2020-01-01'}
     ],
+    'Spyder':[
+        {content: '3', start: '2016-09-24', end: '2019-03-01', py2:true},
+        {content: '4', start: '2019-02-01', end: '2020-02-01', py2:true},
```

Comment:
This should be `start: 2019-01-15`, `end: 2019-12-15`

Reply:
Following the above reasoning, we would want to shift the end date to at least the start date of the next version, 2019-12-31.